compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/BaseComposeScene.skiko.kt

```diff
@@ -57,9 +57,11 @@ import kotlinx.coroutines.withContext
 @OptIn(InternalComposeUiApi::class)
 internal abstract class BaseComposeScene(
     coroutineContext: CoroutineContext,
-    private val invalidate: () -> Unit,
+    private val invalidate: (canRenderImmediately: Boolean) -> Unit,
```

Comment:
```suggestion
    private val invalidate: (throttledToVsync: Boolean) -> Unit,
```

Seems the logic is opposite now.

And it has this name in the other places.

Reply:
I was flipping the logic and renaming it and this one was forgotten.
The logic is correct, only the name is wrong.

> And it has this name in the other places.
I only see `canRenderImmediately` here.

Reply:
> I only see canRenderImmediately here.

I meant that it had `throttledToVsync` in the other places